MM2RandoLib/MM2RandoLib.csproj

```diff
@@ -1359,9 +1359,9 @@
     </ItemGroup>
     <ItemGroup>
       <PackageReference Include="js65.interop" Version="1.0.0-alpha3" />
-      <PackageReference Include="Microsoft.ClearScript.V8.Native.linux-x64" Version="7.4.5" Condition="$([MSBuild]::IsOsPlatform('Linux'))" />
-      <PackageReference Include="Microsoft.ClearScript.V8.Native.osx-arm64" Version="7.4.5" Condition="$([MSBuild]::IsOsPlatform('OSX'))" />
-      <PackageReference Include="Microsoft.ClearScript.V8.Native.win-x64" Version="7.4.5" Condition="$([MSBuild]::IsOsPlatform('Windows'))" />
+      <PackageReference Include="Microsoft.ClearScript.V8.Native.linux-x64" Version="7.4.5" />
```

Comment:
It seems like because it's built all at once and then packaged for different platforms this may not be set. Either that or IsOsPlatform is returning the server platform, not the target platform. A better solution is $(RuntimeIdentifier), but that isn't set for AnyCPU configuration. Thus we need to include all three in the build and hopefully the packager will separate them.